SU-HSDO/suhumsci
# SHS-5169: Color Band update

# READY FOR REVIEW

## Summary
- Creates new link and text fields for Color Band component
- Updates label and character length validation for Color Band text area field
- Moves existing Color Band text area field content to new text field if it is less than 105 characters

## Steps to Test
1. Visit the Color Band page in [Colorful](https://hs-colorful.suhumsci.loc/components/color-bands) or Traditional
2. Edit the various color bands to include all variations (colors, all fields filled, some fields, one field, no fields)
3. Ensure that they match the [designs provided:](https://www.figma.com/proto/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?type=design&node-id=578-6681&t=hI3763D2wQgFrEdn-0&scaling=min-zoom&page-id=575%3A4764&hide-ui=1) and hover states also match
4. Go back and follow the same steps for the other theme (Colorful or Traditional)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
